_config.yml

```diff
@@ -7,16 +7,16 @@ logo: /assets/img/logo_sgds.png
 url: https://www.designsystem.tech.gov.sg
 
 # Site data
-sgds-version: 1.3.22
+sgds-version: 2.0.0
 report-vulnerability: "https://tech.gov.sg/report_vulnerability"
 contact-us: "https://form.gov.sg/forms/govtech/5bc440ac507956000fdbae91"
 feedback-form: "https://form.gov.sg/#!/forms/govtech/5bd12c6668dbe7000fff9ace"
 request-component: "https://form.gov.sg/6136ea11658bea0012fc9e80"
 search_placeholder: "What are you looking for?"
-react-storybook: https://1-0-0-dev.dg2jutx9gylat.amplifyapp.com/
-github-link: https://github.com/GovTechSG/sgds/tree/2.0.0-dev
+react-storybook: https://react.designsystem.tech.gov.sg
+github-link: https://github.com/GovTechSG/sgds/tree/v2
```

Comment:
change to https://github.com/GovTechSG/sgds/tree/v2dev

Reply:
I've placed in the `/v2/` so we wont have to update during launch

Reply:
oh okies noted